recipes/robin-hood-hashing/config.yml

```diff
@@ -1,15 +1,7 @@
 versions:
-  "3.7.0":
```

Comment:
Why drop the older versions? 

I liked you PRs with summaries:)

Reply:
explained above ;) https://github.com/conan-io/conan-center-index/pull/16617#issuecomment-1475312013

Reply:
Can you take a look at https://github.com/SpaceIm/conan-center-index/pull/15? merge it if it looks ok to you and the rest of the PR lgtm :)

Reply:
> explained above ;) [#16617 (comment)](https://github.com/conan-io/conan-center-index/pull/16617#issuecomment-1475312013)

Derp thank you I absolutely missed that :)